pegasus/rake/test.rake

```diff
@@ -1,8 +1,10 @@
 ENV['RACK_ENV'] = 'test'
 
+ENV['HONEYBADGER_LOGGING_LEVEL'] = 'error'
+
 require 'rake/testtask'
 require_relative '../test/fixtures/fake_dashboard.rb'
-Rake::TestTask.new
+Rake::TestTask.new.tap{|x| x.warning = false}
```

Comment:
This is a separate change to filter ruby warnings from rake tests (Rake 11 changed the default for `warnings` from `false` to `true` which started spamming our test runs). I just included this change in this PR since it was low-risk and touched the same file.